GOVERNANCE.md

```diff
@@ -85,11 +82,15 @@ Note that the PGB may permit the formation or continuation of a TSC that does no
 
 ### TSC funding
 
-The OASIS Ethereum Open Project supports all of its TSCs.  However, it could happen that a TSC would like additional resources, beyond those provided by default by OASIS, to support its initiatives.  In this case, a TSC may seek additional funding for its projects in one of two ways:
```

Comment:
We should be consistent with the project name, either "Ethereum OASIS" (or "the Ethereum OASIS Open Project" in the first instance).

Reply:
This was a typo.  Good catch.